scripts/koji/koji_build.py

```diff
@@ -105,10 +131,12 @@ def local_branch(branch):
         # prev_branch is empty when the head was detached
         subprocess.check_call(['git', 'checkout', prev_branch or commit])
 
+
```

Comment:
Doubling newlines is not a rule we chose to enforce at the moment.

Reply:
ok i can revert this, fyi flake8 complained about that